configure.ac

```
@@ -1348,6 +1348,33 @@ else
   JAVAC="$JAVACBIN"
 fi
 
+# Check Java version: we require Java 9 or later for Doxygen tests.
+if test -n "$JAVAC"; then
+    AC_MSG_CHECKING(if java version is 9 or greater)
+    javac_version=`$JAVAC -version 2>&1`
```

```suggestion
    javac_version=`"$JAVAC" -version 2>&1`
```
To fix Appveyor output:
```
configure: WARNING: unknown format for Java version returned by C:/Program Files (x86)/Java/jdk1.8.0/bin/javac (./configure: line 8548: C:/Program: No such file or directory)
```

---

Thanks, I missed this one.